diff --git a/media/libstagefright/ACodec.cpp b/media/libstagefright/ACodec.cpp
index d42fc5acf4..84d374e9ff 100644
--- a/media/libstagefright/ACodec.cpp
+++ b/media/libstagefright/ACodec.cpp
@@ -66,7 +66,6 @@
 #include "include/SecureBuffer.h"
 #include "include/SharedMemoryBuffer.h"
 #include <media/stagefright/omx/OMXUtils.h>
-#include "TableXInit.h"
 
 #include <server_configurable_flags/get_flags.h>
 
@@ -3192,49 +3191,6 @@ status_t ACodec::setupAC4Codec(
 
     def.nChannels = numChannels;
     def.nSampleRate = sampleRate;
-    
-    OMX_AUDIO_PARAM_ANDROID_AC4TBL tbl;
-    InitTblOMXParams(&tbl);
-
-    TableXInit *A_OBJ = new TableXInit(AC4_TABLE_SEC_FRS_CODE,
-                                       AC4_TABLE_SEC_FRS_MASK_VAL);
-
-    TableXInit *B_OBJ = new TableXInit(AC4_TABLE_SEC_MDD_MAX_FRAM,
-                                       AC4_TABLE_SEC_MMF_MASK_VAL);
-
-    TableXInit *C_OBJ = new TableXInit(AC4_TABLE_SEC_MDD_MAX_INST,
-                                       AC4_TABLE_SEC_MMI_MASK_VAL);
-
-    A_OBJ->init();
-    B_OBJ->init();
-    C_OBJ->init();
-
-    tbl.seedA = A_OBJ->getSeed();
-    tbl.seedB = B_OBJ->getSeed();
-    tbl.seedC = C_OBJ->getSeed();
-
-    tbl.sizeA = A_OBJ->getSize();
-    tbl.sizeB = B_OBJ->getSize();
-    tbl.sizeC = C_OBJ->getSize();
-
-    tbl.idA = A_OBJ->getTableID();
-    tbl.idB = B_OBJ->getTableID();
-    tbl.idC = C_OBJ->getTableID();
-
-    tbl.maskA = A_OBJ->getMaskVal();
-    tbl.maskB = B_OBJ->getMaskVal();
-    tbl.maskC = C_OBJ->getMaskVal();
-
-    memcpy (tbl.bufferA, A_OBJ->getBuffer(), LUT_BUFFER_SIZE);
-    memcpy (tbl.bufferB, B_OBJ->getBuffer(), TABLE_B_C_U8_SZ);
-    memcpy (tbl.bufferC, C_OBJ->getBuffer(), TABLE_B_C_U8_SZ);
-
-    mOMXNode->setParameter(
-            (OMX_INDEXTYPE)OMX_IndexParamAudioAndroidAc4Tbl, &tbl, sizeof(tbl));
-
-    delete A_OBJ;
-    delete B_OBJ;
-    delete C_OBJ;
 
     return mOMXNode->setParameter(
             (OMX_INDEXTYPE)OMX_IndexParamAudioAndroidAc4, &def, sizeof(def));
